### ISSUE TITLE
Support for Wild card SSL Certificate

### ISSUE THREAD

**Issue by adarshmadrecha** [carlalexander/debops-wordpress#209]
When creating a new Multisite WordPress installation, it would be awesome if when the user is selecting "use subdomains" option, `debops-wordpress` automatically installs wildcard SSL from `letsencrypt`.

This is with reference to below wiki guide.
https://github.com/carlalexander/debops-wordpress/wiki/How-to-create-a-multisite-network#configuring-a-multisite-network-to-use-subdomains

**Comment by wpbullet** [comment]
@adarshmadrecha Let's Encrypt WIldcard SSL requires DNS records but after that is done I'm sure @carlalexander can add an option for this

**Comment by carlalexander** [comment]
Yeah, I need to look it. I haven't had to use it yet, but it might not be a bad idea to have that be the default behaviour. 😄
